### PR TITLE
Release 1.11.3

### PR DESCRIPTION
**Pull request by migingreen**
**Number-poll & -guess**
- Omit 4 digit number formatting
- Redesign slider label
- Various small style adjustments

[Q-Item](https://q.st-staging.nzz.ch/item/e9046b127bd99afc9cd208b94d6e34da) for testing in stage

### CODE REVIEW

File: views/NumberGuess.html [hunk 1]
```diff
@@ -2,14 +2,14 @@
   <div class="q-quiz-input-range-container">
     <div class="q-quiz-input-range-position-label-container">
       <div
-        class="
-          q-quiz-input-range-position-label
-          s-font-note s-font-note--strong s-font-note--tabularnums
-          s-color-primary-7
-        "
+        class="q-quiz-input-range-position-label"
```
Comment (manuelroth) [inline views/NumberGuess.html:5]:
```suggestion
        class="q-quiz-input-range-position-label s-color-primary-5"
```

File: styles_src/numberGuess.scss [hunk 2]
```diff
@@ -127,7 +129,22 @@ $q-quiz-result__number-guess-visual__stats-graphic-height: 60px;
...
+.q-quiz-input-range-container {
+  margin-bottom: $basis-margin * 6;
+}
+
+.q-quiz-input-range-position-label-container {
+  margin: 0 14px (9px + ($basis-margin * 3));
+}
+
 .q-quiz-input-range-position-label {
+  background-color: $nzz-blue-medium;
```
Comment (manuelroth) [inline styles_src/numberGuess.scss:141]:
```suggestion
  background-color: currentColor;
```

File: styles_src/numberGuess.scss [hunk 3]
```diff
@@ -1,5 +1,7 @@
 $q-quiz-result__number-guess-visual__stats-graphic-top: 55px;
 $q-quiz-result__number-guess-visual__stats-graphic-height: 60px;
+$nzz-blue-medium: #2c32bd;
```
Comment (manuelroth) [inline styles_src/numberGuess.scss:3]:
```suggestion
```

Comment (manuelroth) [reply styles_src/numberGuess.scss:3]: This hard coded color value could be avoided by using the sophie-color class `s-color-primary-5` and setting `background-color: currentColor` on the element.

Comment (migingreen) [reply styles_src/numberGuess.scss]: Thanks, I did look for the color in the visuals styleguide and in the NZZ Design System but couldn't find a css class reference.

Comment (manuelroth) [reply styles_src/numberGuess.scss]: Yes its true. The primary colors are unfortunately not in the styleguide. Would be a good idea to add them. It the moment you can find them here: https://github.com/nzzdev/sophie-color/blob/master/vars/general.json#L15

File: views/NumberPoll.html [hunk 4]
```diff
@@ -2,14 +2,14 @@
   <div class="q-quiz-input-range-container">
     <div class="q-quiz-input-range-position-label-container">
       <div
-        class="
-          q-quiz-input-range-position-label
-          s-font-note s-font-note--strong s-font-note--tabularnums
-          s-color-primary-7
-        "
+        class="q-quiz-input-range-position-label"
```
[the remark this answers is not in the document]
Comment (migingreen) [reply views/NumberPoll.html:5]: Exactly, numberpoll uses the same stylesheet, so only the html has to be updated.

**Review by manuelroth — APPROVED** [review]

**Pull request merged**